init-wifi.h

```diff
@@ -9,19 +11,37 @@ struct WifiConfig {
 };
 
 const char *wifiName = "/wifiConfig.json";
-WifiConfig wifiConfig;
+static WifiConfig wifiConfig;
```

Comment:
when do we put static modifier? eg qrConfig has no static modifier

Reply:
This was done to ensure that qrConfig is set to 0 when initialized.
Need to be tested without static keyword. If works - this keyword will be removed.

Reply:
right - it works for qrConfig without static
and this is static wifiConfig

so are static or global variables set to 0 by default?

Reply:
Works correct without static keyword. 
This keyword is removed.